docs/concepts/endpoint-structure/request-response-structure.md

```diff
@@ -12,11 +12,68 @@ From this base endpoint, we can assemble almost all paths described in the subse
 
 ## Request format
 
-In Shopware 6, the request body has to be JSON encoded. Always set the `Content-Type` header to `application/json` if you're sending data to the API.
+The request format for Shopware 6 store API's must follow the below format:
 
-It's mandatory to use type-safe values, e.g. if the API expects an integer value, you're required to provide an actual integer. If you're using a Date field, make sure to use an ISO 8601 compatible date format.
+* The request body has to be JSON encoded.
 
-```javascript
+* It is mandatory to use type-safe values, e.g., if the API expects an integer value, you are required to provide an actual integer.
+
+* When using a Date field, make sure to use an ISO 8601 compatible date format.
+
+### Request URL
+
+The Admin API can be separated into two general sets of endpoints
+
+* Generic entity endpoints
+
+* Specific interaction/configuration endpoints
+
+Both fulfill specific cases which are outlined below.
+
+#### Generic entity endpoints
+
+Provide CRUD functionalities on **all entities available** in Shopware. Go to the [Entity Reference](../resources/entity-reference.md) section to see a list of all available entities and their structure.
+
+Furthermore, these endpoints can be divided into **read** and **write** operations. To learn more about those, please head to the corresponding guide on [Reading Entities](../guides/reading-entities.md) and [Writing Entities](../guides/writing-entities/README.md). These endpoints are generic, since they are entirely based on the entities definitions and contain no additional business logic besides data validation.
```

Comment:
```suggestion
Furthermore, these endpoints can be divided into **read** and **write** operations. To learn more about those, please head to the corresponding guide on [Reading Entities](reading-entities.md) and [Writing Entities](writing-entities/README.md). These endpoints are generic, since they are entirely based on the entity definitions and contain no additional business logic besides data validation.
```